pkg/trtl/honu_test.go

```diff
@@ -269,26 +264,21 @@ func (s *trtlTestSuite) TestDelete() {
 		},
 	})
 	require.NoError(err)
-	// TODO update os.Getenv to use test fixtures in sc-2098
-	pid := os.Getenv("TRTL_REPLICA_PID")
-	expectedPID, err := strconv.Atoi(pid)
-	require.NoError(err)
-	expectedRegion := os.Getenv("TRTL_REPLICA_REGION")
-	owner := bytes.Join([][]byte{[]byte(pid), []byte(expectedRegion)}, []byte(":"))
+	owner := bytes.Join([][]byte{[]byte(fmt.Sprint(metaPID)), []byte(metaRegion)}, []byte(":"))
```

Comment:
The convention I try to follow is to use `strconv.Itoa` when converting integers to strings, and `fmt.Sprint` when something needs to be formatted for printing output, although I'm not sure what the underlying difference actually is.